### PR TITLE
endpoint: use PropertyCEP{Owner,Name} as CEP owner/name if set

### PR DESCRIPTION
**Pull request by jibi**
see individual commits

### CODE REVIEW

**Review by aanm — APPROVED** [review]

**Review by aanm — APPROVED** [review]

File: pkg/endpoint/endpoint.go [hunk 1]
```diff
@@ -85,6 +85,12 @@ const (
 	// PropertySkipBPFRegeneration will mark the endpoint to skip ebpf
 	// regeneration.
 	PropertySkipBPFRegeneration = "property-skip-bpf-regeneration"
+
+	// PropertyCEPOwner will be able to store the CEP owner for this endpoint.
+	PropertyCEPOwner = "property-cep-owner"
+
+	// PropertyCEPName contains the CEP name for this endpoint.
+	PropertyCEPName = "property-cep-name-wrapper"
```
Comment (pippolo84) [inline pkg/endpoint/endpoint.go]: Just a curiosity: why `property-cep-name-wrapper` and not `property-cep-name`?

Comment (jibi) [reply pkg/endpoint/endpoint.go]: good shout, just a leftover I think :+1:

**Review by pippolo84 — APPROVED** [review]
LGTM, thanks! :100:

**Pull request merged**